js/mcqModel.js

```diff
@@ -168,10 +168,59 @@ define([
                 _selectedItems: [],
                 _isAtLeastOneCorrectSelection: false
             });
+        },
+
+        getInteractionObject: function() {
+            var interactions = {
+                correctResponsesPattern: [],
+                choices: []
+            };
+
+            interactions.choices = this.get('_items').map(function(item) {
+                return {
+                    id: (item._index + 1).toString(),
+                    description: item.text
```

Comment:
i always get stuck with where the line get's drawn between xapi api and adapt provision.